apps/etl-func/avro/message.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "message",
+  "type": "record",
+  "namespace": "dto",
+  "doc": "Kafka JS schema for cosmos api container 'messages'",
+  "fields": [
+    {
+      "name": "schema_version",
+      "type": "int"
+    },
+    {
+      "name": "op",
+      "type": {
+        "type": "enum",
+        "name": "MessageCrudOperation",
+        "symbols": ["CREATE"]
+      }
+    },
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name": "sender_service_id",
+      "type": "string"
+    },
+    {
+      "name": "sender_user_id",
+      "type": "string"
+    },
+    {
+      "name": "is_pending",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "content_subject",
+      "type": "string"
+    },
+    {
+      "name": "content_type",
+      "type": [
+        "null",
+        {
+          "type": "enum",
+          "name": "MessageContentType",
+          "symbols": ["GENERIC", "PAYMENT", "EU_COVID_CERT", "LEGAL", "PN"]
+        }
+      ],
+      "default": null
```

Comment:
Can you explain the meaning of these `content_type` values?
What's the difference between `null` and `GENERIC`?
As per requirements, we have to include also `PAGOPA_RECEIPT`

Reply:
@lucacavallaro in the[ avro specification](https://avro.apache.org/docs/1.12.0/specification/) we got `doc` to put a simple description of the field, I was thinking about adding it to all the fields, what's your take?

Reply:
👍

Reply:
Take a look here https://github.com/pagopa/io-messages/pull/40/commits/efe0b201c2734690a92eede44b2e3d47e28d7f07